DEVELOPERS.md

```diff
@@ -52,111 +54,70 @@ To contribute code to Supabase, you must fork the [Supabase Repository](https://
-   cd apps/reference
+   npm run dev # start all the applications
    ```
 
-   Go to the [Supabase Studio](https://app.supabase.com) directory
-
-   ```sh
-   cd studio
-   ```
-
-1. Install npm/yarn dependencies:
-
-   npm
-
-   ```sh
-   npm install
-   ```
-
-   or with yarn
-
-   ```sh
-   yarn install
-   ```
-
-## Start a development server
-
-To debug code and to see your changes in real time, it is often useful to have a local HTTP server. Click one of the three links below to choose which development server you want to start.
-
-- [Supabase Website](#supabase-website)
-- [Supabase Docs](#supabase-docs)
-- [Supabase Studio](#supabase-studio)
+Then visit, and edit, any of the following sites:
 
-### Supabase Website
+Site | Directory | Scope name | Description | Local development server
+---- | --------- | ---------- | ----------- | ------------------------
+[supabase.com](https://supabase.com) | `/apps/www` | www | The main website | http://localhost:3000
+[app.supabase.com](https://app.supabase.com) | `/studio` | studio | Studio dashboard | http://localhost:8082
+[POC] Next.js based Docs site | `/apps/temp-docs` | temp-docs | Guides and Reference (Next.js based) | http://localhost:3001  
+[POC] Community forum | `/apps/temp-community-forum` | forum | GitHub Discussions | http://localhost:3002
+[POC] DEV articles site | `/apps/temp-community-tutorials` | tutorials | DEV articles (which community members can write) | http://localhost:3003
 
-The website is moving to a new monorepo setup. See the [Monorepo](#monorepo) section below.
+The following sites are temporarily not run using turborepo, and must be run independently.
 
-### Supabase Docs
+Site | Directory | Scope name | Description | Local development server
+---- | --------- | ---------- | ----------- | ------------------------ 
+[supabase.com/docs](https://supabase.com/docs) | `/apps/reference` | N/A | Guides and Reference (Currently not in Turborepo) | http://localhost:3010/docs
 
-1. Build development server
+For the docs website, please refer to this [developers guide](https://github.com/supabase/supabase/tree/master/apps/reference/DEVELOPERS.md).
 
-   npm
+#### Shared components
 
-   ```sh
-   npm run build
-   ```
-
-   or with yarn
-
-   ```sh
-   yarn build
-   ```
-
-1. Start development server
-
-   npm
-
-   ```sh
-   npm run start
-   ```
-
-   or with yarn
-
-   ```sh
-   yarn start
-   ```
+The monorepo has a set of shared components under `/packages`:
 
-1. Access the local server in your web browser at http://localhost:3010/docs.
+- `/packages/common`: Common React code, shared between all sites.
+- `/packages/config`: All shared config
+- `/packages/spec`: Generates documentation using spec files.
+- `/packages/tsconfig`: Shared Typescript settings
 
-### Supabase Studio
+#### Installing packages
 
-1. Start development server
+Installing a package with NPM workspaces requires you to add the `-w` flag to tell NPM which workspace you want to install into.
```

Comment:
"You should install all dependencies for any app from the root directory, not from each app"  ?